account-kit/rn-signer/android/src/main/java/com/accountkit/reactnativesigner/NativeTEKStamperModule.kt

```diff
@@ -75,9 +86,21 @@ class NativeTEKStamperModule(reactContext: ReactApplicationContext) :
         EncryptedSharedPreferences.PrefValueEncryptionScheme.AES256_GCM
     )
 
+    // This allows us to do the HPKE decryption of the bundle
+    private val hpkeParams = HpkeParameters.builder()
+        .setKemId(HpkeParameters.KemId.DHKEM_P256_HKDF_SHA256)
+        .setKdfId(HpkeParameters.KdfId.HKDF_SHA256)
+        .setAeadId(HpkeParameters.AeadId.AES_256_GCM)
+        .setVariant(HpkeParameters.Variant.NO_PREFIX)
+        .build()
+
     init {
         TinkConfig.register()
 
+        if (Security.getProvider(BouncyCastleProvider.PROVIDER_NAME).javaClass != BouncyCastleProvider::class.java) {
+            Security.removeProvider(BouncyCastleProvider.PROVIDER_NAME)
+        }
+
```

Comment:
This was due to the fact that android adds its own bouncy castle implementation that doesn't include `EC`